sound_byte/lib/pages/friendList.dart

```diff
@@ -4,4 +4,99 @@ import 'package:sound_byte/services/authentication.dart';
 import 'package:sound_byte/pages/userProfile.dart';
 import 'package:sound_byte/pages/friendProfile.dart';
 import 'package:sound_byte/pages/friendScreen.dart';
+import 'package:sound_byte/pages/user.dart';
 
+class FriendList extends StatelessWidget{
+  User user;
+  List<String> user_friends;
+  int number_of_friends;
+  String user_id;
+
+  FriendList (User user) {
+    this.user = user;
+    this.user_friends = user.friends;
+    this.number_of_friends = user.friends.length();
+    this.user_id = user.userId;
+  }
+
+  FriendList.nullFriendList() {
+    this.user = new User("Sound Byte", 
+                        "Soundbyte@gmail.com", 
+                        "12345");
+    this.user_friends = new List<String>();
+    this.number_of_friends = 0;
+    this.user_id = "12345";
+  }
+
+  FriendList.full(User user) {
+    this.user = user;
+    this.user_friends = user.friends;
+    this.number_of_friends = this.user_friends.length();
+    this.user_id = user.userId;
+  }
+
+  bool isNull () {
```

Comment:
this function is never called, and doesn't seem to actually check a null friendsList correctly

Reply:
I took this method out. This was originally created because I did not have a FriendsListNotNull method initially so I had to check if a user object was null.